src/usr/local/buildpack/prepare/golang.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+require_root
+
```

Comment:
Should already be be enforced
```suggestion
```

Reply:
yes, i think it's easier to check if the `golang` folder at `/usr/local/golang` exists than checking if all deb packages are created.

On some tools we make some more configs on prepare, so i think the simplest check is to validate the folder exists.


I recently had the idea to simplify folder handling.
* install all tools to `/usr/local` (even user install)
* allow root group writing to eg `/usr/local/golang` and `/usr/local/bin`
* should make prepare and install checks more easy.
WDYT?

Reply:
> yes, i think it's easier to check if the `golang` folder at `/usr/local/golang` exists than checking if all deb packages are created.
>
> On some tools we make some more configs on prepare, so i think the simplest check is to validate the folder exists.

Fair point

> I recently had the idea to simplify folder handling.
> 
>     * install all tools to `/usr/local` (even user install)
> 
>     * allow root group writing to eg `/usr/local/golang` and `/usr/local/bin`
> 
>     * should make prepare and install checks more easy.
>       WDYT?

This would indeed make it easier.
We can handle root installs with the root group and make all tools owned by the 1000 user.

Reply:
We should make the unified install directory before this PR then.

Reply:
tool versions installed via root should not be changeble by user. so only the directory should be writable for new  files